docs/source/faq.rst

```diff
@@ -795,10 +793,7 @@ And each user block will be followed by a pagebreak, except the last user.
         return angularParser(tag);
     }
     const doc = new Docxtemplater();
-    doc.setOptions({
-        parser: parser
-    });
-    doc.loadZip(zip);
+    const doc = new Docxtemplater(zip, {parser: parser,});
```

Comment:
Can you also add the little change of the `docx` to `doc`

Reply:
Sure.
Done in [this](https://github.com/shadab14meb346/docxtemplater/commit/c4844b46d0a09fc4a88bca50422d375f95b7d1cc) commit.